includes/annotationcommand_adapter.inc

```diff
@@ -0,0 +1,798 @@
+<?php
+
+/**
+ * @file
+ * annotationcommand_adapter.inc
+ */
+
+use Consolidation\AnnotatedCommand\CommandFileDiscovery;
+use Consolidation\AnnotatedCommand\AnnotatedCommandFactory;
+use Consolidation\AnnotatedCommand\CommandProcessor;
+use Consolidation\AnnotatedCommand\Hooks\HookManager;
+use Consolidation\OutputFormatters\FormatterManager;
+use Consolidation\OutputFormatters\Options\FormatterOptions;
+use Symfony\Component\Console\Input\ArrayInput;
+use Consolidation\AnnotatedCommand\AnnotationData;
+use Consolidation\AnnotatedCommand\CommandData;
+use Drush\Command\DrushInputAdapter;
+use Drush\Command\DrushOutputAdapter;
+
+use Symfony\Component\Console\Output\ConsoleOutput;
+
+/**
+ * Cache the command file discovery object. Drush 9 uses dependency injection.
+ *
+ * @return CommandFileDiscovery
+ */
+function annotationcommand_adapter_get_discovery() {
+  static $discovery;
+  if (!isset($discovery)) {
+    $discovery = new CommandFileDiscovery();
+  }
+  return $discovery;
+}
+
+/**
+ * Initialize and cache the command factory. Drush 9 uses dependency injection.
```

Comment:
I'll fix up the comment. This code will use more DI in the near future, but does not right now.